official/vision/beta/projects/yolo/ops/anchor.py

```diff
@@ -0,0 +1,383 @@
+# Copyright 2021 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Yolo Anchor labler."""
+import tensorflow as tf
+from tensorflow.python.ops.gen_math_ops import maximum, minimum
+from official.vision.beta.projects.yolo.ops import box_ops
+from official.vision.beta.projects.yolo.ops import preprocessing_ops
+from official.vision.beta.projects.yolo.ops import loss_utils
+
+
+def get_best_anchor(y_true,
+                    anchors,
+                    stride,
+                    width=1,
+                    height=1,
+                    iou_thresh=0.25,
+                    best_match_only=False, 
+                    use_tie_breaker=True):
+  """
+  get the correct anchor that is assoiciated with each box using IOU
```

Comment:
Fix doc string, move this line up, and add period at the end.

Reply:
fixed